packages/integrations/cloudflare/README.md

```diff
@@ -87,7 +87,7 @@ It's then possible to update the preview script in your `package.json` to `"prev
 
 ## Access to the Cloudflare runtime
 
-You can access all the Cloudflare bindings and environment variables from Astro components and API routes through `Astro.locals`.
+You can access all the Cloudflare bindings and environment variables from Astro components and API routes through `Astro.locals`. Usually you need to use [Wrangler](https://github.com/cloudflare/workers-sdk) to be able to access the runtime.
```

Comment:
```suggestion
You can opt in to access all the Cloudflare bindings and environment variables from Astro components and API routes through `Astro.locals`. This replaces using [Wrangler](https://github.com/cloudflare/workers-sdk) to access the runtime.
```
I think this needs to be clearer, because the above phrase is a bit sophisticated English usage that requires familiarity with the two parts of that common structure, and only implies the (important) second half. It *could* potentially be interpreted as "Usually, you WILL (present tense! You will! this is our guidance!) use Wrangler to do this" NOT as you intended, "Usually you'd use Wrangler, **but now you don't have to!**") 

You could also add on to the existing sentence something that is explicitly clear, 

> ... to access the runtime, but configuring `cloudflare.runtime` will allow you to use Astro's dev server instead."

Reply:
While I concur with your comment, the technical implementation requires an additional PR to align with it. For now, I've removed the reference to wrangler in this section, as the other part of the Readme already contains all the necessary information.